mediafile.py

```diff
@@ -1774,8 +1774,12 @@ def as_dict(self):
         ListStorageStyle('GENRE'),
         ASFStorageStyle('WM/Genre'),
     )
-    genre = genres.single_field()
-
+    genre = MediaField(
+        MP3StorageStyle('genre'),
```

Comment:
this is actually wrong. should be `TCON`, if genre should be a "simple string version" of genres plural.